index.js

```diff
@@ -1,118 +1,45 @@
-const BooksCollection = [
-  {
-    title: 'In Search of Lost Time',
-    author: 'Marcel Proust',
-  },
-  {
-    title: 'Ulysses',
-    author: 'James Joyce',
-  },
-  {
-    title: 'Don Quixote',
-    author: 'Miguel de Cervantes',
-  },
-  {
-    title: 'One Hundred Years of Solitude',
-    author: 'Gabriel Garcia Marquez',
-  },
-  {
-    title: 'The Great Gatsby',
-    author: 'F. Scott Fitzgerald',
-  },
-];
+import Book from './modules/bookClass.js';
+import BookCRUD from './modules/bookCrudClass.js';
+import Interface from './modules/interfaceClass.js';
+import { onlyDisplayBook, onlyDisplayInputBook, onlyDisplayContact } from './modules/onlyDisplay.js';
+import { DateTime } from './node_modules/luxon/src/luxon.js';
 
-const bookContainer = document.querySelector('.book-container');
+document.addEventListener('DOMContentLoaded', Interface.renderBooks);
 
-function loadBook(book) {
-  const divBook = document.createElement('div');
-  divBook.className = 'book';
-  const pBookTitle = document.createElement('p');
-  pBookTitle.className = 'book-title';
-  pBookTitle.innerText = `Book Title: "${book.title}"`;
-  const pBookAuthor = document.createElement('p');
-  pBookAuthor.className = 'book-author';
-  pBookAuthor.innerText = `Book Author: "${book.author}"`;
-  const buttonBookRemove = document.createElement('button')
-  buttonBookRemove.className = 'book-remove';
-  buttonBookRemove.innerText = 'Remove';
-  buttonBookRemove.setAttribute('onclick', 'removeBook(this)');
-  const hrBookSeparation = document.createElement('hr');
-  divBook.appendChild(pBookTitle)
-  divBook.appendChild(pBookAuthor);
-  divBook.appendChild(pBookAuthor);
-  divBook.appendChild(buttonBookRemove);
-  divBook.appendChild(hrBookSeparation);
-  bookContainer.appendChild(divBook);
-  // const bookContent = `
-  //   <div class="book">
-  //     <p class="book-title">Book Title: "${book.title}"</p>
-  //     <p class="book-author">Author By: ${book.author}</p>
-  //     <button class="book-remove">Remove</button>
-  //     <hr class="book-separation">
-  //   </div>
-  // `;
-  // bookContainer.innerHTML += bookContent;
-};
+const addForm = document.getElementById('add-form');
+const titleInput = document.getElementById('book-title');
+const authorInput = document.getElementById('book-author');
 
-function loadBooksInformation() {
-  while (bookContainer.hasChildNodes()) {
-    bookContainer.removeChild(bookContainer.firstChild);
+addForm.addEventListener('submit', (e) => {
+  e.preventDefault();
+  const title = titleInput.value;
+  const author = authorInput.value;
+  const book = new Book(title, author);
```

Comment:
- [ ] Please your app generates an error at line 17 when a user tries to add a new book identifying book as a constructor.
**Check the error message below**
![class](https://user-images.githubusercontent.com/55596970/227929158-8322f00d-4009-45d4-ba01-ef79b9e010b9.png)
Please properly refactor your code.

Reply:
has fixes for add book